app/uk/gov/hmrc/taxenrolmentassignmentfrontend/connectors/EACDConnector.scala

```diff
@@ -131,4 +124,28 @@ class EACDConnector @Inject()(httpClient: HttpClient,
       )
   }
 
+  def queryEnrolmentsAssignedToUser(userId: String)
+                                   (implicit hc: HeaderCarrier,
+                                    ec: ExecutionContext): TEAFResult[Option[UserEnrolmentsListResponse]] = EitherT {
+    val url = s"${appConfig.EACD_BASE_URL}/enrolment-store/users/$userId/enrolments?type=principal"
+
+    httpClient
+      .GET[HttpResponse](url)
+      .map(
+        httpResponse =>
+          httpResponse.status match {
+            case OK         => Right(Some(httpResponse.json.as[UserEnrolmentsListResponse]))
+            case NO_CONTENT => Right(None)
+            case status =>
+              logger.logEvent(
+                logES2ErrorFromEACD(
+                  userId,
+                  status,
+                  httpResponse.body
+                )
+              )
```

Comment:
Not really sure why this has been added? The method above would give you all the credentials that have a PT enrolment?